pkg/trace/api/api.go

```diff
@@ -213,14 +238,13 @@ func (r *HTTPReceiver) Start() {
 		return
 	}
 
-	timeout := getConfiguredRequestTimeoutDuration(r.conf)
 	httpLogger := log.NewThrottled(5, 10*time.Second) // limit to 5 messages every 10 seconds
 	r.server = &http.Server{
-		ReadTimeout:  timeout,
-		WriteTimeout: timeout,
```

Comment:
Is there any downside to removing the `WriteTimeout`? If not, do we still need the `ReadTimeout`? Will the context deadline handle both?

Reply:
I assume that the `Handler` code only gets called after the request has been fully read. Since we set the context deadline in the `Handler`, it doesn't protect us if the request takes a long time to be read and we still need `ReadTimeout`.

On the other hand, `WriteTimeout` shouldn't be needed if our code handles not taking too long to write the response, which should now do.

That's based on how I assume `http.Server` is implemented and the docs.